### DISCUSSION TITLE
Sketch hochladen bei Punktgeometrie

### DISCUSSION THREAD

**Discussion by marko-jer** [e-netze/webgis-community#168] [General]
Beim erfassen von Linien- und Flächengeometrien besteht die Möglichkeit des "Sketch hochladen" :
![image](https://github.com/gis-eni/webgis-forum/assets/1842500/cf2ae3a2-beae-49e6-8c34-a783d5d352ef)
Bei Punktgeometrien fehlt diese Option:
![image](https://github.com/gis-eni/webgis-forum/assets/1842500/22e066c5-acf0-4734-9993-9af23d01d294)

Es wird für das Hochladen von GPX-Waypoints gebraucht.

**Comment by gis-eni** [comment]
Ja das wurde absichtlich weggelassen. Die Frage war für mich, wer hat Files mit einzelnen Punkten, die er hochladen möchte. 
Bei PGX stell ich mir die Frage, wie viele Waypoints sind da drin? Man muss ja für jeden Punkt das File neue hochladen und einen Punkt auswählen...
Wäre das nicht Hochladen ins Redlining die besser option?

Oder beschreib einmal den genaueren Anwendungsfall. Da gibt es bestimmt eine elegantere Lösung

**Reply by marko-jer** [reply]
Anwendungsfall ist einen Aufnahme von Punkten mit einem GPS-Gerät und das Einspielen der Ergebnisses GPX-Waypoints ins webGIS zum Zwecke der Abfrage der Grundstücke.

Erfassen in eine Feature Class - Da hast du recht, man muss jeden Punkt einzeln verspeichern --> kann sehr aufwändig werden.

Redlining-Ebene ist da nicht wirklich die Lösung, da mit den Redlining-Symbolen keine Abfrage möglich ist, könnte aber zur Darstellung (Dokumentation als pdf-Ausdruck) dienen,
**wenn**
auch bei der Abfrage/Identifizieren das Laden aller Waypoints eines Files für eine Multipoint-Abfrage (AGS kann das) ermöglicht wird.

Nach genauer Klärung wird nun ein direktes Erfassen mit der Field Maps-App durchgeführt. 
Somit hat diese Anfrage momentan keine hohe Priorität.
